src/client/src/redux/actions/fetch-brainmates-action.js

```diff
@@ -0,0 +1,27 @@
+import actionTypes from "./constants";
+import BrainmatesApiService from "../../services/brainmates-api-service";
+
+const fetchBrainmatesRequestAction = () => ({
+  type: actionTypes.FETCH_BRAINMATES_REQUEST,
+});
```

Comment:
Why you use the action but you don't assign a state telling that the brainmates are loading?